client/tests/unit/CosmologyForm.spec.js

```diff
@@ -80,6 +80,17 @@ describe('Mounted CosmologyForm', () => {
       expect(emitted.updateCosmo.length).toBeGreaterThan(prevCount);
       prevCount = emitted.updateCosmo.length;
     }
+    const { model } = wrapper.vm;
+    for (const key of Object.keys(model)) {
+      if (typeof model[key] === 'number') {
+        prevCount = emitted.updateCosmo.length;
+        wrapper.vm.$data.model[key] += 1;
+        await localVue.nextTick();
+        await localVue.nextTick();
+        expect(emitted.updateCosmo.length).toBeGreaterThan(prevCount);
+        prevCount = emitted.updateCosmo.length;
+      }
+    }
   });
```

Comment:
It would be good if this was in a separate test just to further hone in on where things might fail.